modules/compute/vm-instance/variables.tf

```diff
@@ -392,3 +392,13 @@ variable "allocate_ip" {
   })
   default = null
 }
+
+variable "disable_automatic_updates" {
```

Comment:
My preference would be for `run_automatic_package_updates: true` to be the default.
or maybe better `allow_automatic_package_updates: true`

This comes from our previous experience with naming variables with the prefix `disable_`. It can be confusing with the double negative when reading the variable documentation or as seen in blueprints if we were ever to change the default value.

I realize that the actual action we are taking is to disable the feature, and that using `disable_` may be a more accurate statement, but I believe the potential user confusion is more important.

When reading documentation we would either have
`disable_automatic_updates` (default `false`) OR
`allow_automatic_package_updates` (default `true`)
to me the second is clearer

This would be my vote. I am fine to be out-voted by other reviewers.

Reply:
I've updated all references to be `allow_automatic_updates` and specified in the variable description that this is specifically for system packages.